pkg/volume/local/local.go

```diff
@@ -107,6 +107,7 @@ func (plugin *localVolumePlugin) NewMounter(spec *volume.Spec, pod *v1.Pod, _ vo
 			mounter:    plugin.host.GetMounter(),
 			plugin:     plugin,
 			globalPath: volumeSource.Path,
+                        MetricsProvider: volume.NewMetricsDu(volumeSource.Path),
```

Comment:
Hm I would use StatFS then since most of the other plugins use that, and it'll be more reliable than parsing output from a command.